cpp/ql/src/Likely Bugs/Format/TooManyFormatArguments.ql

```diff
@@ -9,6 +9,7 @@
  * @id cpp/too-many-format-arguments
  * @tags reliability
  *       correctness
+ *       security
```

Comment:
We split this query out from `cpp/wrong-number-format-arguments` and gave it severity `recommendation` because we believe it doesn't have security implications.